framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2530,4 +2540,28 @@ private void printGitProperties() {
             System.out.println("IOException while reading git.properties: " + e.getMessage());
         }
     }
+
+    /** Extract the version of the Checker Framework */
```

Comment:
You need to document that this can return null.  Under what circumstances can it do so?  Should it throw an exception rather than returning null?